scripts/windows/SetDeploymentTokenScript.ps1

```diff
@@ -0,0 +1,35 @@
+$Token = "";
+
+if ([string]::IsNullOrEmpty($Token))
+{
+    Write-Host "Deployment token not provided. Exiting."
+    Exit
+}
+
+$zorusRegistry = "Registry::HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Zorus Inc.\Archon Agent"
```

Comment:
Do we need to worry about 32 bit OS support?

Reply:
good eye. i wonder how powershell reacts to this compared to dotnet. gonna validate this concern

Reply:
updated to use the powershell function that should have been provided to me upon googling accessing registry stuff the first time